rocsparse/PKGBUILD

```diff
@@ -1,39 +1,27 @@
 # Maintainer: Markus Näther <naetherm@informatik.uni-freiburg.de>
 pkgname=rocsparse
-_pkgver=3.3.0
-pkgver="$_pkgver"
-pkgrel=1
-pkgdesc="Next generation SPARSE implementation for ROCm platform."
+pkgver=3.3.0
+pkgrel=2
+pkgdesc='BLAS for sparse computation on top of ROCm'
 arch=('x86_64')
-url="https://github.com/ROCmSoftwarePlatform/rocSPARSE"
-license=('custom:NCSAOSL')
-depends=('hcc' 'hip')
-makedepends=('cmake' 'hcc' 'python2' 'rocminfo')
-source=("https://github.com/ROCmSoftwarePlatform/rocSPARSE/archive/rocm-$_pkgver.tar.gz")
+url='https://rocmdocs.amd.com/en/latest/ROCm_Libraries/ROCm_Libraries.html#rocsparse'
+license=('MIT')
+depends=('hip-hcc' 'rocprim')
+makedepends=('cmake' 'hcc' 'git')
+_git='https://github.com/ROCmSoftwarePlatform/rocSPARSE'
+source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz")
 sha256sums=('04122317b21df69709a11b164b42e0f21f7fc745c2e28daeecb99ab01e7a8de3')
 
 build() {
-  mkdir -p "$srcdir/build"
-  cd "$srcdir/build"
-
-  # Tensile library needs python to be python2
-  export PATH="$srcdir:$PATH"
-  [[ -e "$srcdir/python" ]] || ln -s /usr/bin/python2 "$srcdir/python"
-
-  # fix broken build with stack protection
-  export CFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CFLAGS")"
-  export CXXFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CXXFLAGS")"
-  export CPPFLAGS="$(sed -e 's/-fstack-protector-strong//' <<< "$CPPFLAGS")"
-
-  # compile with HCC
-  export CXX="/opt/rocm/hcc/bin/hcc"
+  mkdir -p build
+  cd build
 
-  # TODO: fix librocsparse.so, it contains references to $srcdir
-  cmake -DCMAKE_BUILD_TYPE=Release \
-        -DCMAKE_INSTALL_PREFIX=/opt/rocm/rocsparse \
+  #TODO: librocsparse.so contains references to $srcdir
```

Comment:
Let's also get rid of this TODO, you can make an issue for it instead.

Reply:
Done.